src/llama_stack/providers/inline/agents/meta_reference/responses/types.py

```diff
@@ -160,6 +161,8 @@ class ChatCompletionContext(BaseModel):
     temperature: float | None
     response_format: OpenAIResponseFormatParam
     tool_context: ToolContext | None
+    responses_tool_choice: OpenAIResponseInputToolChoice | None = None
+    chat_tool_choice: str | dict[str, Any] | None = None
```

Comment:
lots of different types here in this union, Is this going to be hard to enforce?

Reply:
not sure I follow, by setting this union we are enforcing the types no?

Reply:
ack, updated accordingly